core/src/com/unciv/logic/map/UnitMovementAlgorithms.kt

```diff
@@ -219,6 +219,7 @@ class UnitMovementAlgorithms(val unit:MapUnit) {
         if(unit.type.isAirUnit()){ // they move differently from all other units
             unit.action=null
             unit.removeFromTile()
+            unit.isTransported = false // it has left the carrier by own means
```

Comment:
This assignment to `isTransported` will be overwritten by `putInTile`.

I don't think we need to clear this flag here, because `putInTile` will set it correctly.

Reply:
There are 2 scenarios when air unit gets into the city: 
1) either by own mean from the carrier or another city
2) or being transported when carrier docks in the city.
If the city is full, the first scenario should deny landing, but the second scenario should work. This line is needed to not allow reach the full city from the another city.